code/modules/jobs/job_types/roguetown/nobility/lady.dm

```diff
@@ -72,6 +72,6 @@
 	new_role = "Servant"
 	recruitment_faction = "Servants"
 	recruitment_message = "Serve the crown, %RECRUIT!"
-	accept_message = "FOR THE CROWN!"
-	refuse_message = "I refuse."
+	accept_message = "FOR SERVITUDE!"
```

Comment:
What?

Reply:
Yeah, I don't like this change.

Reply:
Fine, I changed it to "FOR SOVEREIGNTY!"